File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldSignature.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnCaptureValueKind;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldSignature extends StudioFieldEditable
{
  @Nullable
  public Boolean captureLocation;

  @Nullable
  public Boolean captureTime;

  @Nullable
  public Boolean captureUser;

  @Nullable
  public EnumDefnCaptureValueKind[] showCapturedValuesOnAside;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldSignature.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnCaptureValueKind

interface StudioFieldSignature : StudioFieldEditable {
    val captureLocation: Boolean?
    val captureTime: Boolean?
    val captureUser: Boolean?
    val showCapturedValuesOnAside: List<EnumDefnCaptureValueKind>?
}
